CRM/Civioffice/Form/DocumentUpload.php

```diff
@@ -63,7 +63,7 @@ public function buildQuickForm()
                 if ($file) {
                     unlink($file);
                     CRM_Core_Session::setStatus(
-                        E::ts("File '%1' deleted.", [1 => basename($file)]),
+                        E::ts("File '%1' deleted", [1 => basename($file)]),
```

Comment:
This should end in a full stop, ideally it would be a full sentence: *File %1 has been deleted.*

Reply:
Changed.